daemon/delete.go

```diff
@@ -102,8 +102,10 @@ func (daemon *Daemon) cleanupContainer(container *container.Container, forceRemo
 	// Save container state to disk. So that if error happens before
 	// container meta file got removed from disk, then a restart of
 	// docker should not make a dead container alive.
-	if err := container.ToDiskLocking(); err != nil {
-		logrus.Errorf("Error saving dying container to disk: %v", err)
+	if _, err := os.Stat(container.Root); err == nil {
```

Comment:
@mlaventure 
Sorry I am not able to follow. How is this (`GetRootResourcePath()`) related.
`ToDiskLocking()` calls `ToDisk` which basically goes and execute this line.
https://github.com/docker/docker/blob/master/container/container.go#L134

Since `pth` (container.Root) does not exist, `os.Create(pth)` blows up.